### ISSUE TITLE
Channel won't close (transaction failed)

### ISSUE THREAD

**Issue by rethore** [raiden-network/raiden#1138]
## Problem Definition

I've tried to reproduce the API calls described in readthedocs. While most of them worked, I couldn't close a channel (and therefore also settle it afterwards). 

When I made the following API request [1]

`PATCH /api/1/channels/<CHANNEL ID>`, with payload `{"state":"closed"}`

 I received:

```json
{
 "partner_address": "0x0075703bf50aebe6755844c0c9b28cdb8f730933",
 "balance": 1050,
 "reveal_timeout": 10,
 "settle_timeout": 100,
 "state": "opened",
 "channel_address": "0xcf312d3743e09ccb7630c185b80d65747479bb6a",
 "token_address": "0x706ffe280b3ab30719212a8b6f4a1b2ac00f0862"
}
``` 

Which indicates that the channel state is still open.

Furthermore, I checked the transaction initiated in the parity node on etherscan [2]. The transaction status is `TxReceipt Status:Failed`

[1] API requests:
``` In [142]: patch(raiden1 + '/api/1/channels/' + BET_chn, json={"state":"closed"})```
https://github.com/WeOU/energycollective/blob/master/jupyter/play%20with%20raiden.ipynb

[2] Failed transaction: https://ropsten.etherscan.io/tx/0x7574ac8a1327316166550e63835bc253c1d9a2f233f47ecbd3aa2775e4e3cc93

### System Description
**System**: Docker compose [3,4,5], inspired from [raiden/docker/Dockerfile](https://github.com/raiden-network/raiden/blob/master/docker/Dockerfile)
**Raiden version**: commit version: 6be7cc1117be41642fed138e50a3a27c6167e6f5
**Parity version**: stable-release

[3] Docker compose used to start the two Raiden nodes:
https://github.com/WeOU/energycollective/blob/master/docker/docker-compose.yml

[4] Dockerfile for raiden nodes
https://github.com/WeOU/energycollective/blob/master/docker/raiden_webui.Dockerfile

[5] Raiden settings file:
https://github.com/WeOU/energycollective/blob/master/docker/settings.py

**Comment by hackaugusto** [comment]
Same as #1090 and #1109 .

This should be fixed by #1141 , if the bug persists please open a new issue.

**Comment by rethore** [comment]
I can confirm that #1141 fixed this bug. Thanks!
